src/containers/NotificationList/NotificationList.test.tsx

```diff
@@ -67,19 +73,33 @@ test('It should load notifications', async () => {
 
 test('click on forward arrrow', async () => {
   render(notifications);
+
+  await waitFor(() => {
+    expect(screen.getByTestId('loading')).toBeInTheDocument();
+  });
+
   await waitFor(() => {
-    const arrow = screen.getAllByTestId('tooltip');
-    fireEvent.click(arrow[0]);
+    expect(screen.getByText('Notifications')).toBeInTheDocument();
+  });
+
+  const arrowButtons = screen.getAllByTestId('ArrowForwardIcon');
+
+  arrowButtons.forEach(async (button) => {
+    fireEvent.click(button);
+
+    await waitFor(() => {
+      expect(mockedUsedNavigate).toHaveBeenCalled();
+    });
   });
 });
 
 test('it should show copy text and view option on clicking entity ', async () => {
   const { getByTestId, getByText } = render(notifications);
   await waitFor(() => {
-    const entityMenu = screen.getByTestId('NotificationRowMenu');
-    expect(entityMenu).toBeInTheDocument();
+    const entityMenu = screen.getAllByTestId('NotificationRowMenu');
```

Comment:
Not sure how Multiple menu's are created here. Did we change anything in the list?

Reply:
since i changes the mock here to cover all cases, there are multiple components with this test id